server/api/project_apis.py

```diff
@@ -440,3 +443,128 @@ def get(self, project_id: int):
             error_msg = f'Project Summary GET - unhandled error: {str(e)}'
             current_app.logger.critical(error_msg)
             return {"error": error_msg}, 500
+
+
+class TaskAnnotationsAPI(Resource):
+    def get(self, project_id: int, annotation_type: str = None):
```

Comment:
I couldn't get the GET or POST calls to work successfully without a parameter. If that is a blocking point for now, we can drop it and work on it later when broadening the capabilities of the task annotations (such as the property fields to load in the project).